README.md

```diff
@@ -381,6 +386,11 @@ python3 DeepSpeech.py --n_hidden 2048 --checkpoint_dir path/to/checkpoint/folder
 
 Note: the released models were trained with `--n_hidden 2048`, so you need to use that same value when initializing from the release models.
 
+## Data Augmentation or Pertubation
+
+Online Augmentation is a useful technique to for more generalization of machine learning models. Thus, a pre-processing pipeline with various augmentation techniques on audio data and also on spectrogram. 
+Full Documentation `util/augumentation/README.md`. By default augmentation is not enabled, to use this option use flag `--audio_augmentation_config_file="${pwd}/util/augmentation/conf/audio_augmentation.json` for audio augmentation and `--spectrum_augmentation_config_file="${pwd}/util/augmentation/conf/spectrum_augmentation.json`
```

Comment:
nit: Full documentation at

pwd is a command, not an environment variable. "${pwd}" just expands to nothing, which creates a path that points to `/util/augmentation`. Did you test this? Also, you don't need to prepend `pwd` to paths like this, relative paths are relative to `pwd` by default anyway.